rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -358,9 +358,14 @@ class DSCTechniqueWriter(
                 ) ( e =>
                   IOError(s"Could not find dsc Technique '${technique.name}' in path ${basePath}/${techniquePath}", Some(e))
                 )
+      // Powershell files needs to have a BOM added at the beginning of all files
+      contentWithBom : List[Byte] =
+        // Bom, three bytes: EF BB BF
+        239.toByte :: 187.toByte :: 191.toByte  ::
+        content.getBytes(StandardCharsets.UTF_8).toList
       files <-  execute {
                   Files.createDirectories(path.getParent)
-                  Files.write(path, content.getBytes(StandardCharsets.UTF_8))
+                  Files.write(path, contentWithBom.toArray)
```

Comment:
Why not using utf_8_sig ?
https://docs.python.org/2.5/lib/module-encodings.utf-8-sig.html